mappings/net/minecraft/client/util/math/Vector4f.mapping

```diff
@@ -24,7 +26,9 @@ CLASS net/minecraft/class_1162 net/minecraft/client/util/math/Vector4f
 		ARG 4 z
 		ARG 5 w
 	METHOD method_4953 getX ()F
-	METHOD method_4954 multiplyXyz (Lnet/minecraft/class_1160;)V
-		ARG 1 other
+	METHOD method_4954 multiply (Lnet/minecraft/class_1160;)V
+		ARG 1 vector
```

Comment:
Is multiplying x by x1, y by y1, and z by z1 of the vector a multiplication?

Reply:
... yes. What would make that _not_ a multiplication?

Reply:
A vector multiplication means a dot or cross product. This is neither.

On a side note, this 4-d vector serves like a https://en.wikipedia.org/wiki/Homogeneous_coordinates for a 3d space.

Reply:
Alrighty... Is there a term relating to multiplying vectors by component then?

Reply:
Yes, "inner product".

Reply:
https://en.m.wikipedia.org/wiki/Hadamard_product_(matrices), but I would prefer `multiplyComponentwise`.

Reply:
Hm. I was hoping there was a name I was just not remembering (like dotProduct, scalarProduct?). `multiplyComponentWise` looks like it will work.

Reply:
> multiplyComponentWise 

Lowercase `w`. "[componentwise](https://en.wiktionary.org/wiki/componentwise)" is an adjective. "component wise" means a component that's somehow wise, and in the wrong word order.

> was hoping there was a name

There are three. Read the first sentence (or even the title of the link itself!) of the Wikipedia article above. "Hadamard product", "Schur product", and "entrywise product".